src/main.go

```diff
@@ -106,7 +109,7 @@ func main() {
 			conf.Refresh()
 			p := time.Now()
 			for _, r := range conf.Config.Routes {
-				hash := URLHash(r)
+				hash := URLHash(r, "")
```

Comment:
Why do we pass an empty string here, if its deliberate we need to mention that in the comments

Reply:
Sure.